### ISSUE TITLE
MkReleaseTest.java:197-199: Release.Smart().prerelease()...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1390]
The puzzle `1363-762aa75e` from #1363 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/939d24d94a900936560157d1d137f5f793e411ac/src/test/java/com/jcabi/github/mock/MkReleaseTest.java#L197-L199

The puzzle was created by amihaiemil on 18-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#1390](https://github.com/jcabi/jcabi-github/issues/1390) is now in scope, role is `DEV`

**Comment by llorllale** [comment]
@0crat waiting on merge of #1397

**Comment by 0crat** [comment]
> @0crat waiting on merge of #1397  ([here](https://github.com/jcabi/jcabi-github/issues/1390#issuecomment-383594293))

@llorllale The impediment for [#1390](https://github.com/jcabi/jcabi-github/issues/1390) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1363-762aa75e` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1390](https://github.com/jcabi/jcabi-github/issues/1390) is now out of scope

**Comment by llorllale** [comment]
@amihaiemil any idea when this one will be released?
